.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Could you add a `group:` definition to this? With a `group`, Dependabot will send us 1 PR for multiple updates instead of 1 PR per update, which is a lot easier to review 🙂 

Example here:

https://github.com/sigstore/sigstore-python/blob/f71c86178894fe60c4ae582426bdcf4007743835/.github/dependabot.yml#L26-L29

Reply:
Done in 4126e952ffdfc4a5a6e308ae30b5465aefeeb7c0